pservice/CMakeLists.txt

```diff
@@ -65,7 +65,8 @@ SET(SGX_INCLUDE "${SGX_SDK}/include")
 SET(SGX_SSL_INCLUDE "${SGX_SSL}/include")
 SET(SGX_ENCLAVE_INCLUDE "${SGX_SDK}/include"
                         "${SGX_SDK}/include/tlibc"
-                        "${SGX_SDK}/include/libcxx")
+                        "${SGX_SDK}/include/libcxx"
+                        "${SGX_SSL}/include")
 
```

Comment:
why are these necessary? we shouldn't be including anything directly from SSL. It should only be included through common.

Reply:
I think we can skip these changes. PDO builds fine without them.

Reply:
OK. But it is strange that it reports error when building eservice if I don't add the SSG_SSL/include to include path:
[ 35%] Building CXX object lib/libpdo_enclave/CMakeFiles/libpdo-enclave.dir/contract_enclave.cpp.o
In file included from /home/agent/WASM/private-data-objects/eservice/lib/libpdo_enclave/contract_code.cpp:22:0:
/home/agent/WASM/private-data-objects/eservice/../common/crypto/crypto.h:18:10: fatal error: openssl/sha.h: No such file or directory
 #include <openssl/sha.h>

Reply:
Is is up to you. I can manually modify the makefile to make it build success.